packages/webapp/vite.config.ts

```diff
@@ -37,4 +38,9 @@ export default defineConfig({
   server: {
     port: 3000,
   },
+  resolve: {
+    alias: {
+      '@shared': path.resolve(__dirname, '../shared'),
```

Comment:
Added this alias so that it's easier to import from the shared folder in the webapp files.